configuration/authentication/azure-entra.md

```diff
@@ -0,0 +1,54 @@
+---
+description: How to configure Azure Entra Authentication
+---
+
+# Azure Entra
+
+## Overview
+Kafbat UI comes with an OAuth Callback Handler for 
+[Azure Event Hubs for Kafka](https://learn.microsoft.com/en-us/azure/developer/java/spring-framework/migrate-kafka-to-passwordless-connection?tabs=azure-portal%2Csign-in-azure-cli%2Cjava-kafka%2Capp-service%2Cassign-role-service-connector).
+
+The Callback Handler uses the 
+[Default Azure Credential](https://learn.microsoft.com/en-us/azure/developer/java/sdk/identity-azure-hosted-auth#default-azure-credential) for authentication, 
+providing support for all commonly used Entra credentials.
+
+Pre-requisites:
+1. An Event Hub Namespace in the standard, premium or dedicated tiers.
+2. An Event Hub.
+3. Entra Credentials for an Entra Identity with a role assignment allowing 
+   sending to and receiving from Event Hub, such as Azure Event Hubs Data Owner.
+
+## Examples
+
+Please replace `<EVENT HUB NAMESPACE NAME>` with the name of your Event Hub namespace.
+
+### Docker
+
+[Default Azure Credential](https://learn.microsoft.com/en-us/azure/developer/java/sdk/identity-azure-hosted-auth#default-azure-credential) 
```

Comment:
nit - I think this is implying that I need to somehow authenticate using any of the supported default az cred methods.  the most common likely being az login - it may be worth adding some narrative here indicating that any of the supported methods will work, for example az login - this is my opinion and should not block proceeding

Reply:
lgtm